add UI

VadimRomanyuk opened the pull request.

YanaYaroshevich commented on Task3/UI/styles.css:
```diff
@@ -0,0 +1,198 @@
+  width: auto;
+}
+
+#footer
```
старайтесь использовать только классы

YanaYaroshevich commented on Task3/UI/styles.css:
```diff
@@ -0,0 +1,198 @@
+
+#footer
+{   position: fixed;
+    left: 0; bottom: 0;
```
одна строка - один стиль

YanaYaroshevich commented on Task3/UI/styles.css:
```diff
@@ -0,0 +1,198 @@
+#footer
+{   position: fixed;
+    left: 0; bottom: 0;
+    font-size : 65%;
```
не используйте проценты для размера шрифта

YanaYaroshevich commented on Task3/UI/styles.css:
```diff
@@ -0,0 +1,198 @@
+    float: left;
+}
+
+.headPost
```
разделяйте слова в названиях с помощью "-"

YanaYaroshevich commented on Task3/UI/index.html:
```diff
@@ -0,0 +1,272 @@
+</head>
+<body>
+ <header>
+    <p class="header" style="position: absolute; left:45%; top:0%">
```
все стили в css

YanaYaroshevich commented on Task3/UI/index.html:
```diff
@@ -0,0 +1,272 @@
+ <header>
+    <p class="header" style="position: absolute; left:45%; top:0%">
+        PhotoLover</p>
+        <a href = "#" class = "buttonedit" style="position: absolute; top:4%;right: 5%;">
```
лишние пробелы

YanaYaroshevich commented on Task3/UI/index.html:
```diff
@@ -0,0 +1,272 @@
+                                    <div class="elements">
+                                        <img src="img/fone.jpg"  class="fone">
+                                        <img src="img/pencilred.jpg" class="setting">
+                                        <tr><td colspan="2" ><br /><input type="image" src="img/like.jpg"  name="bottom" value="" class="like"></td></tr>
```
теги таблицы?

YanaYaroshevich commented on Task3/UI/index.html:
```diff
@@ -0,0 +1,272 @@
+                                        <img src="img/fone.jpg"  class="fone">
+                                        <img src="img/pencilred.jpg" class="setting">
+                                        <tr><td colspan="2" ><br /><input type="image" src="img/like.jpg"  name="bottom" value="" class="like"></td></tr>
+                                        <img src="img/trash.jpg" class="delete">
```
это должна быть кнопка

YanaYaroshevich commented on Task3/UI/index.html:
```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
```
подравняйте разметку
сейчас выглядит плохо на любом разрешении
было бы лучше переделать

YanaYaroshevich commented on Task3/UI/styles.css:
```diff
@@ -0,0 +1,198 @@
+body{
```
желательно, чтобы была какая-то конкретная цель для использования каждого стиля
лучше не верстать "на глазок"
смотрите на страницу как на совокупность блоков, которые нужно на ней разместить, не выйдя за границы

VadimRomanyuk commented:
Здравствуйте, я тут полностью свой сайт переделал. Можете , пожалуйста, пересмотреть мое третье задание? Только я заливал без mockups, потому что сайт будет  слегка видоизменен.  Мне нужно новый mockups создавать или можно перезалить старый? Спасибо

________________________________
От: Yana <notifications@github.com>
Отправлено: 4 марта 2019 г. 18:46:04
Кому: VadimRomanyuk/BSU-practice
Копия: Романюк Вадим Н.; State change
Тема: Re: [VadimRomanyuk/BSU-practice] add UI (#2) [html][faked-from]


@YanaYaroshevich commented on this pull request.

________________________________

In Task3/UI/styles.css<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262114174>:

> @@ -0,0 +1,198 @@

+body{

+    background-color: #b38df7;

+}

+header{

+  background:  white;

+  height: 40%  ;

+  width: auto;

+}

+

+#footer



старайтесь использовать только классы

________________________________

In Task3/UI/styles.css<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262114257>:

> @@ -0,0 +1,198 @@

+body{

+    background-color: #b38df7;

+}

+header{

+  background:  white;

+  height: 40%  ;

+  width: auto;

+}

+

+#footer

+{   position: fixed;

+    left: 0; bottom: 0;



одна строка - один стиль

________________________________

In Task3/UI/styles.css<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262114349>:

> @@ -0,0 +1,198 @@

+body{

+    background-color: #b38df7;

+}

+header{

+  background:  white;

+  height: 40%  ;

+  width: auto;

+}

+

+#footer

+{   position: fixed;

+    left: 0; bottom: 0;

+    font-size : 65%;



не используйте проценты для размера шрифта

________________________________

In Task3/UI/styles.css<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262114586>:

> +  margin-left: 1%;

+  list-style-type: none;

+}

+.hashtegs

+{

+    font-family: cursive;

+    color: blue;

+    font-size: 150%;

+    margin-top: 52%;

+    margin-right: auto;

+    margin-left: 1%;

+    margin-bottom: 40px;

+    float: left;

+}

+

+.headPost



разделяйте слова в названиях с помощью "-"

________________________________

In Task3/UI/index.html<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262114825>:

> @@ -0,0 +1,272 @@

+<!DOCTYPE html>

+<html lang="en">

+<head>

+    <meta charset="UTF-8">

+    <meta name="viewport" content="width=device-width, initial-scale=1.0">

+    <meta http-equiv="X-UA-Compatible" content="ie=edge">

+    <link href="styles.css" rel="stylesheet" type="text/css">

+    <title>Document</title>

+</head>

+<body>

+ <header>

+    <p class="header" style="position: absolute; left:45%; top:0%">



все стили в css

________________________________

In Task3/UI/index.html<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262114927>:

> @@ -0,0 +1,272 @@

+<!DOCTYPE html>

+<html lang="en">

+<head>

+    <meta charset="UTF-8">

+    <meta name="viewport" content="width=device-width, initial-scale=1.0">

+    <meta http-equiv="X-UA-Compatible" content="ie=edge">

+    <link href="styles.css" rel="stylesheet" type="text/css">

+    <title>Document</title>

+</head>

+<body>

+ <header>

+    <p class="header" style="position: absolute; left:45%; top:0%">

+        PhotoLover</p>

+        <a href = "#" class = "buttonedit" style="position: absolute; top:4%;right: 5%;">



лишние пробелы

________________________________

In Task3/UI/index.html<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262115549>:

> +                    </section>

+

+                    <a href = "#" class = "buttonedit" style="position: fixed; bottom:24%;right: 3%; width:10%; height: 10%; text-align: center">

+                            Delete old Photopost</a>

+

+                            <div class="post">

+                                    <div class="headPost">

+                                        <img src="img/profile.jpg" alt=""  class="userphoto">

+                                        <div class="username">Winnie</div>

+                                        <div class="datapost"> 10.10.2018  10:08  </div>

+                                    </div>

+                                        <img src="img/afe90d7c3d792a05dca741d5f7544529.jpg" class = "photo" >

+                                    <div class="elements">

+                                        <img src="img/fone.jpg"  class="fone">

+                                        <img src="img/pencilred.jpg" class="setting">

+                                        <tr><td colspan="2" ><br /><input type="image" src="img/like.jpg"  name="bottom" value="" class="like"></td></tr>



теги таблицы?

________________________________

In Task3/UI/index.html<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262115962>:

> +

+                    <a href = "#" class = "buttonedit" style="position: fixed; bottom:24%;right: 3%; width:10%; height: 10%; text-align: center">

+                            Delete old Photopost</a>

+

+                            <div class="post">

+                                    <div class="headPost">

+                                        <img src="img/profile.jpg" alt=""  class="userphoto">

+                                        <div class="username">Winnie</div>

+                                        <div class="datapost"> 10.10.2018  10:08  </div>

+                                    </div>

+                                        <img src="img/afe90d7c3d792a05dca741d5f7544529.jpg" class = "photo" >

+                                    <div class="elements">

+                                        <img src="img/fone.jpg"  class="fone">

+                                        <img src="img/pencilred.jpg" class="setting">

+                                        <tr><td colspan="2" ><br /><input type="image" src="img/like.jpg"  name="bottom" value="" class="like"></td></tr>

+                                        <img src="img/trash.jpg" class="delete">



это должна быть кнопка

________________________________

In Task3/UI/index.html<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262116537>:

> @@ -0,0 +1,272 @@

+<!DOCTYPE html>



подравняйте разметку
сейчас выглядит плохо на любом разрешении
было бы лучше переделать

________________________________

In Task3/UI/styles.css<https://github.com/VadimRomanyuk/BSU-practice/pull/2#discussion_r262116800>:

> @@ -0,0 +1,198 @@

+body{



желательно, чтобы была какая-то конкретная цель для использования каждого стиля
лучше не верстать "на глазок"
смотрите на страницу как на совокупность блоков, которые нужно на ней разместить, не выйдя за границы

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://github.com/VadimRomanyuk/BSU-practice/pull/2#pullrequestreview-210208935>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AtbW0oLpTOS1fo5Sbhnp-ZHjVUDtkvyRks5vTT-8gaJpZM4ba45z>.
